### PR TITLE
Update documentation index

### PR DESCRIPTION
**Pull request by ben-polinsky**
- Re-Add documentation index. Previously this was integrated from a different repo.
- Bump Bemetalsmith version.

### CODE REVIEW

File: docs/config/.updated.json [hunk 1]
```diff
@@ -0,0 +1,5537 @@
+{
+  "index.html": {
```
Comment (calebmshafer) [inline docs/config/.updated.json:2]: Did this used to be in another repo or was it dynamically generated all the time?

Comment (williamkbentley) [reply docs/config/.updated.json:2]: I thought .updated.json was created and uploaded as a build artifact

Comment (ben-polinsky) [reply docs/config/.updated.json:2]: Are we talking about the index.html referenced in the .update.json file, or the json file itself?

Comment (williamkbentley) [reply docs/config/.updated.json:2]: I interpreted Caleb's comment as a question on .updated.json file as a whole

Comment (ben-polinsky) [reply docs/config/.updated.json]: Must be the way I am building the docs locally. Removed.

**Review by calebmshafer — APPROVED** [review]
LGTM

**Comment by ben-polinsky** [comment]
@mergifyio backport release/3.5.x

**Pull request merged**